### PR TITLE
simplify main.js

### PR DESCRIPTION
**Pull request by ghost**
Change the "if" statment to two simple statements.

### CODE REVIEW

**Comment by XhmikosR** [comment]
Not sure if this is an improvement.

**Comment by XhmikosR** [comment]
@Trott thoughts? I personally don't think this is an improvement.

**Comment by Trott** [comment]
> @Trott thoughts? I personally don't think this is an improvement.

I appreciate the desire to improve the code but I think the existing code is more maintainable. It's easier to understand, at least to me. 

@MaledongGit Is there something in particular about the existing code that makes it problematic that you were hoping to improve?

**Review by nickserv — APPROVED** [review]

**Review by aymen94 — APPROVED** [review]

**Comment by nickserv** [comment]
This change reduces repetition. I find the current code difficult to read as the duplication makes the intent of the code harder to understand. The variable name also makes it easier to understand.

**Review by 3imed-jaberi — APPROVED** [review]

**Review by osk2 — APPROVED** [review]

File: static/js/main.js [hunk 1]
```diff
@@ -25,12 +25,8 @@
...
   langPickerTogglerElement.addEventListener('click', function () {
     langPickerElement.classList.toggle('hidden')
-
-    if (langPickerTogglerElement.getAttribute('aria-expanded') === 'true') {
-      langPickerTogglerElement.setAttribute('aria-expanded', 'false')
-    } else {
-      langPickerTogglerElement.setAttribute('aria-expanded', 'true')
-    }
+    const isAriaExpanded = langPickerTogglerElement.getAttribute('aria-expanded') === 'true'
+    langPickerTogglerElement.setAttribute('aria-expanded', (!isAriaExpanded).toString())
```
Comment (tniessen) [inline static/js/main.js]: I understand @Trott's point, the boolean/string conversions are not pretty. However, if I remember all those questionable DOM / WebIDL conversion specifications correctly, the whole `(...).toString()` is unnecessary:

```suggestion
    langPickerTogglerElement.setAttribute('aria-expanded', !isAriaExpanded)
```

(I luckily haven't had to write frontend JS in a while, so I might be wrong, but it seems to work in FF/IE/Chrome.)

Comment (Trott) [reply static/js/main.js]: > I understand @Trott's point,

Clearly I'm in the minority on this. I'm fine with it landing.

**Comment by XhmikosR** [comment]
This whole thing is like premature optimization. I honestly don't see any real benefit from this patch. The code works fine, and we don't even have to get into if `toString()` is needed :P

**Pull request merged**